scripts/resetdb.sh

```diff
@@ -3,9 +3,9 @@ echo "Completely wipe and reset database 'test'."
 read -p "Are you sure? " -n 1 -r
 if [[ $REPLY =~ ^[Yy]$ ]]
 then
-    mysql -u root -e 'DROP DATABASE IF EXISTS test;'
-    mysql -u root -e 'CREATE DATABASE test;'
-    mysql -u root -e "GRANT ALL ON test.* TO 'test'@'localhost' IDENTIFIED BY 'zaphod';"
-    mysql -u root -D test < storage/mysql/storage.sql
+    mysql -u root "$@" -e 'DROP DATABASE IF EXISTS test;'
```

Comment:
I can see how that would allow for a different password, but does that allow for a different user? Does a later "-u" overrides a previous one?

I would prefer something more explicit, like:
mysql_opts="$@"
if [[ ! "$mysql_opts" || "${mysql_opts##*-u *}" ]]; then
  mysql_opts="-u root $mysql_opts"
fi
mysql "$mysql_opts" ...

Or maybe just add a comment saying -u overrides?